ui/src/app/shared/components/object-editor.tsx

```diff
@@ -121,7 +121,7 @@ export function ObjectEditor<T>({type, value, buttons, onChange}: Props<T>) {
                     options={{
                         readOnly: onChange === null,
                         minimap: {enabled: false},
-                        lineNumbers: 'off',
+                        lineNumbers: 'on',
```

Comment:
Didn't you say in the issue that this could just be removed entirely as it is default `on`? If so, it should be removed then.

Reply:
You are right. This line has been deleted.